### PR TITLE
chore(build): add script for fixing locales from xtm for tacokit

### PR DESCRIPTION
**Pull request by undx**

### CODE REVIEW

File: .jenkins/xtm-deploy.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+echo "Packaging locales"
+mvn -e -B -s .jenkins/settings.xml clean package -pl . -Pi18n-deploy
+if [[ ! $? -eq 0 ]] ; then
+    echo mvn error during xtm packaging
+    exit 123
+fi
+
+echo "Fixing locales for tacokit"
+cd tmp/repository
```
Comment (clesaec) [inline .jenkins/xtm-deploy.sh]: remove "cd tmp/repository"
replace find .
with
 (find tmp/repository -name ... ) on line 14

Comment (undx) [reply .jenkins/xtm-deploy.sh]: No I can't, I need to be in the folder for next operations.

Comment (rmannibucau) [reply .jenkins/xtm-deploy.sh]: side note: "tacokit" does not exist and is likely not what you meant there;)

Comment (undx) [reply .jenkins/xtm-deploy.sh]: right, changed.

File: .jenkins/xtm-deploy.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+echo "Packaging locales"
+mvn -e -B -s .jenkins/settings.xml clean package -pl . -Pi18n-deploy
+if [[ ! $? -eq 0 ]]; then
+  echo   mvn error during xtm packaging
+  exit   123
+fi
+
+echo "Fixing locales to lang only and not lang_COUNTRY"
+cd tmp/repository
+arr=("fr_FR:fr" "ja_JP:ja" "cn_CZ:cz_CN" "en_US:en") && for aa in ${arr[@]}; do
```
Comment (clesaec) [inline .jenkins/xtm-deploy.sh]: mv cn_CZ -> cz_CN ? not cn_CN => cn and cz_CZ => cz ?

Comment (pteyssier) [reply .jenkins/xtm-deploy.sh]: should be zh_CN
so "zh_CN:zh"?

Comment (ypiel-talend) [reply .jenkins/xtm-deploy.sh]: 'cn_CZ' should stay 'cn_CZ' (and not cz_CN) it was voluntary (asked to romain)

**Pull request merged**